CONTRIBUTING.md

```diff
@@ -0,0 +1,152 @@
+# CONTRIBUTING
+
+- [Contributor Guide](#contributor-guide)
+  - [Prerequisites](#prerequisites)
+    - [Create a GitHub account](#create-a-github-account)
+    - [Sign the CLA](#sign-the-cla)
+    - [Code of Conduct](#code-of-conduct)
+    - [Setting up your development environment](#setting-up-your-development-environment)
+  - [Development](#development)
+  - [Testing](#testing)
+    - [Pre-submits and Unit Tests](#pre-submits-and-unit-tests)
+    - [End to End (e2e) Tests](#end-to-end-e2e-tests)
+    - [Linters](#linters)
+  - [Community](#community)
+    - [Communication](#communication)
+
+## Contributor Guide
+
+This guide will help you understand the overall organization of the
+[slsa-github-generator](https://github.com/slsa-framework/slsa-github-generator)
+project, and direct you to the best places to get started contributing. You'll
+be able to pick up issues, write code to fix them, and get your work reviewed
+and merged.
+
+This document is the single source of truth for how to contribute to the code
+base. Feel free to browse the [open issues] and file new ones, all feedback
+is welcome!
+
+### Prerequisites
+
+Before submitting code, you should first complete the following
+prerequisites.
+
+#### Create a GitHub account
+
+Before you get started, you will need to [signup](http://github.com/signup) for a GitHub user account.
+
+#### Sign the CLA
+
+> TODO: is this needed?
+
+Before you can contribute to Kubernetes, you will need to sign the
+[Contributor License Agreement].
+
+#### Code of Conduct
+
+> TODO: Is there a CoC for the OpenSSF that we can point to?
```

Comment:
https://github.com/ossf/wg-best-practices-os-developers/blob/main/code-of-conduct.md

seems like most repos have this one

Reply:
I'll add a link here: https://github.com/slsa-framework/slsa/blob/main/code-of-conduct.md